tools/slo-generator/slo_generator/backends/elasticsearch.py

```diff
@@ -45,46 +44,3 @@ def count(self, timeseries):
             int: Event count.
         """
         raise NotImplementedError()
-
```

Comment:
Why removing this block?

Reply:
I think `yapf` removed the last line

Reply:
`pylint` was complaining, plus it's not the best place to put the example queries, that would fit more in the docs I think